package/instance-manager

```diff
@@ -25,6 +25,16 @@ function enable_tgtd() {
     tgtd -f 2>&1 | tee /var/log/tgtd.log &
 }
 
+function generate_nvme_hostid_and_hostnqn() {
+    mkdir -p /etc/nvme
+
+    local hostnqn=$(nvme gen-hostnqn)
+    # hostnqn is generated from /sys/class/dmi/id/product_uuid according to the implemetation of libnvme
+    echo "$hostnqn" > /etc/nvme/hostnqn
```

Comment:
I assume `/etc/nvme` exists or just make -p to ensure it?

Reply:
`/etc/nvme` doesn't exist because we compile nvme-cli in the `builder` container and copy it into the `release` container.